libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -134,10 +134,14 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     // @Path: ../AP_SBusOut/AP_SBusOut.cpp
     AP_SUBGROUPINFO(sbus, "_SBUS_",  20, SRV_Channels, AP_SBusOut),
 
+    // @Group: _MOCR_
+    // @Path: ../AP_MotionController/AP_MotionController.cpp
+    AP_SUBGROUPINFO(volz, "_MOCR_",  21, SRV_Channels, AP_MotionController),
+
 #if HAL_SUPPORT_RCOUT_SERIAL
     // @Group: _BLH_
     // @Path: ../AP_BLHeli/AP_BLHeli.cpp
-    AP_SUBGROUPINFO(blheli, "_BLH_",  21, SRV_Channels, AP_BLHeli),
+    AP_SUBGROUPINFO(blheli, "_BLH_",  22, SRV_Channels, AP_BLHeli),
```

Comment:
Oh, can't do this!  We can't move the position of existing parameters or they become corrupted.  So the MOCR params should be moved to a free spot normally at the end of the list.